features/unicode-range.yml

```diff
@@ -0,0 +1,7 @@
+name: unicode-range
+description: The `unicode-range` CSS descriptor specifies the characters to be used from a font face, using Unicode codepoints individually, in a range, or with a wildcard match. If the characters aren't present on the page, the font is not downloaded.
```

Comment:
cc: @autonome 

I'm going to do something terrible here which is to lay down a lot of text on how to do something, then ask you to ignore it and throw it out—skip to HOWEVER if you find this unbearable. 😄 

So I'm on a bit of a crusade against "specify" (and "determine") in place of "set", "pick", or "choose." They're a bit like "utilize" — they have a lot of bonus syllables that say the same thing as a more direct alternative ("use"). But this crusade is for a good cause: speaking directly to developers, in a direct and confident way.

Anyway, here's my attempt to revise this. I think not loading is the central point, so I tried to move it to the beginning of the description, where you're more likely to read/skim it.

```suggestion
description: The `unicode-range` CSS descriptor controls whether to load a font, if the characters picked by the descriptor appear in the page. You can choose characters by Unicode code point, code point range, or wildcard pattern.
```

HOWEVER—and I only noticed this after writing most of the above, so forgive me—I think this feature is somewhat needless. It's part of `@font-face` and doesn't meaningfully depart from its overall status. I think we should add the `unicode-range` key to `font-face` and the `font-unicode-range` to the `caniuse` array.

Reply:
Not unbearable at all! And I agree with the merge.